xtask/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "xtask"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Please add a `publish = false` line to avoid accidentally publishing this:

```suggestion
edition = "2021"
publish = false
```

Reply:
You're also very welcome to add a one-line description so people can find out what this is.

Reply:
Have pushed a new PR (https://github.com/google/comprehensive-rust/pull/2707) for this and the other suggestions.